pkg/cache/cache.go

```diff
@@ -148,6 +148,17 @@ func (c *Cache) newClusterQueue(cq *kueue.ClusterQueue) (*ClusterQueue, error) {
 	return cqImpl, nil
 }
 
+func (c *Cache) AdmittedWorkloadsLocalQueue(localQueue *kueue.LocalQueue) int32 {
+	c.Lock()
+	defer c.Unlock()
+	cq, ok := c.clusterQueues[string(localQueue.Spec.ClusterQueue)]
+	if !ok {
+		return 0
```

Comment:
Maybe we should also return error here.

Reply:
Actually, the ClusterQueue might be created after the LocalQueue.

During that time, there can't be any admitted workloads. So 0 is the right value.

Reply:
Ah yes, that's the case. Another approach is 
```
queueObj.Status.AdmittedWorkloads, _ = r.cache.AdmittedWorkloadsLocalQueue(&queueObj)
```

But remaining the same also makes sense to me.

Reply:
if there are no cases of error, I would not add error to the return values